storage.py

```
@@ -0,0 +1,98 @@
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+class SupabaseStorage:
+    def __init__(self):
+        """Initialize Supabase storage client."""
+        self.supabase = create_client(config.SUPABASE_URL, config.SUPABASE_ANON_KEY)
+        self.bucket = config.STORAGE_BUCKET
+        self.public_url = f"{config.SUPABASE_URL}/storage/v1/object/public/{self.bucket}"
+
+    def upload_file(self, file_path: str, object_name: str = None) -> bool:
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)